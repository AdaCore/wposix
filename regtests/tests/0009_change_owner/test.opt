windows DEAD do not have the privilege to change the owner
