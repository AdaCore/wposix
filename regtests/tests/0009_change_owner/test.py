
from test_support import *
import os

gnatmake('changeo')

f=open('check1', 'w')
f.close()

run('changeo')
