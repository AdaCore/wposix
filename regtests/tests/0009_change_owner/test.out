File: check1
 owner: S-1-1-0
 group: S-1-5-32-545
OK, owner has been changed.
OK, group has changed.
